pages/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Amy Cash: Home</title>
+  <link rel="stylesheet" href="../stylesheets/normalize.css">
+  <link rel="stylesheet" href="../stylesheets/main.css">
+</head>
+
+<body>
+  <header>
+    <nav>
+      <div >
+        <ul class="left">
+          <li><a href="index.html" target="_blank">Home</a></li>
+          <li><a href="portfolio.html" target="_blank">Portfolio</a></li>
+          <li><a href="blog.html" target="_blank">Blog</a></li>
+          <li><a href="about.html" target="_blank">About</a></li>
+        </ul>
+      </div>
+
+      <div>
+        <ul  class="right">
+          <li><a href="https://twitter.com/cashmonger?lang=en"><img src=../images/contact-links/twitter.png /></a></li>
+          <li><a href="https://www.linkedin.com/in/amy-cash-0b5305149/"><img src=../images/contact-links/linkedin.png /></a></li>
+          <li><a href="https://www.linkedin.com/in/amy-cash-0b5305149/"><img src=../images/contact-links/email.png /></a></li>
+        </ul>
+      </div>
+
+    </nav>
+  </header>
+
+  <section class="opening-info">
+    <div class="opening-titles">
+      <h1><strong>Amy M. Cash</strong></h1>
+      <h3>Developer of the Future</h3>
+      <h3>Connoisseur of the Past</h3>
+    </div>
+  </section>
+
+
+  <footer>
```

Comment:
We should have this `footer` on every page in the site, similar to the `header` section. This helps to maintain a consistent styling and layout throughout the site.